### PR TITLE
enhancement: binary

### PR DESCRIPTION
**Pull request by mrnvr**
Rename File* to Binary*

### CODE REVIEW

File: dart/lib/model/binary_content_type.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
+part of keyclic_sdk_api.api;
+
+class BinaryContentType {
```
Comment (mrnvr) [inline dart/lib/model/binary_content_type.dart:3]: useless class without any properties

Comment (kevin-lot) [reply dart/lib/model/binary_content_type.dart:3]: This is an issue.
contentType of Binary must be a string.

File: dart/lib/model/binary.dart [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    return Binary(
+      content: json['content'],
+      contentType: json['contentType'],
+      name: json['name'],
+    );
+  }
+
+  String content;
+
+  String contentType;
```
Comment (mrnvr) [inline dart/lib/model/binary.dart:24]: Content type used to be an enum
https://github.com/Keyclic/app-sdk/pull/27/files#diff-0fe0aca7dae4e9219bcb70dec9c327ec4d5d7faf5aa491173d457d5eee7af76bL42-L43

File: swagger.json [hunk 3]
```diff
@@ -16189,6 +16189,50 @@
...
+            "type": "object",
+            "properties": {
+                "content": {
+                    "format": "string",
+                    "type": "string"
+                },
+                "contentType": {
+                    "format": "string",
+                    "type": "string"
+                },
```
Comment (mrnvr) [inline swagger.json:16202]: this used to be an enum
https://github.com/Keyclic/app-sdk/pull/27/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-10cfbe600d3f92e886aea1d00df1367f9899549b1a3235442e5191b66ced7e71L18595-L18611

Comment (kevin-lot) [reply swagger.json:16304]: We will try to get the enum again, with a next swagger definition.

**Pull request merged**